serial_no: "04653450"
cam_name: "camera1"
yaml_url: "package://ximea_camera/calibrations/camera1.yaml"
allocated_bandwidth: 1.0
image_data_format: "XI_RAW8"
